src/tests/report.test.ts

```
@@ -0,0 +1,133 @@
+
+      expect(report).toContain('Evaluation Report');
+      expect(report).toContain('Summary');
+      expect(report).toContain('Detailed Results');
+      expect(report).toContain('Passed');
+      expect(report).toContain('Failed');
+    });
+
+    it('should throw error for unsupported format', () => {
+      const invalidConfig: EvaluationConfig = {
+        ...defaultConfig,
+        outputFormat: 'invalid' as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)